lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -218,6 +218,12 @@ def __init__(self, *args):
 
 The consumer charm can then choose to update its configuration in both situations.
 
+Note that when LokiPushApiConsumer is used, logs will not have any labels automatically added. In
+order to better integrate with the Canonical Observability Stack, you may want to configure your
+software to add Juju topology labels. The
+[observability-libs](https://charmhub.io/observability-libs) library can be used to get topology
+labels in charm code.
```

Comment:
I think a concrete example here could go a long way.

Reply:
As discussed in our meeting, an example could become out of date and there is a link to the docs here.

Reply:
```suggestion
[observability-libs](https://charmhub.io/observability-libs) library can be used to get topology
labels in charm code. See :func:`LogProxyConsumer._scrape_configs` for an example of how
to do this with promtail.
```